lib/phx_tools_web/live/phx_tools_components.ex

```diff
@@ -78,91 +77,39 @@ defmodule PhxToolsWeb.PhxToolsComponents do
                 <%= raw(installation_instruction) %>
               </.os_instructions>
             </div>
-
-            <div class="  mt-2 flex justify-center items-center gap-4 sm:py-4">
-              <.source_code_button source_code_url={@source_code_url} />
-            </div>
-            <div class="-mt-6 ">
-              <p class="text-center text-white font-martian text-xs md:text-sm lg:text-base leading-5 md:leading-6 sm:py-8">
-                <%= phx_tools_description() %>
-              </p>
-            </div>
-
-            <div class=" mb-6 bg-[#110A33] rounded-[4px] shadow-lg text-white shadow-[#2C2650] blur-shadow max-w-[809px]">
-              <div class=" bg-[#2C2650] p-3 rounded-t-[4px] flex sm:flex-col md:flex-row sm:items-start md:items-center sm:space-y-2 md:space-y-0 md:space-x-4 sm:justify-start md:justify-center">
-                <div class="w-full md:w-auto flex justify-center items-center">
-                  <Icons.exclamation_icon />
-                </div>
-                <p class="font-martian text-center sm:text-sm md:text-base lg:text-lg md:py-2">
-                  Unsupported Operating System Detected
-                </p>
+            <div class="justify-center gap-4 mt-2 flex">
+              <div class="sm:grid grid gap-4 sm:py-4">
+                <.source_code_button source_code_url="https://github.com/optimumBA/phx.tools/blob/main/priv/script.sh" />
```

Comment:
Redundant. I can see you've already hard-coded it inside the `source_code_button` component.